### DISCUSSION TITLE
Downgrade Help

### DISCUSSION THREAD

**Discussion by FunnyCorgi** [3kh0/ext-remover#1094] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I was recently able to downgrade to 118. Then, when I tried 112 with my only downgrade usb, it required an update. Can I bypass this by going to 118, then, 115, then 112?

**Comment by helloIlovegimkit** [comment, marked as answer]
You could downgrade or upgrade to any version if your kernal/kernver is 0 and 1 if you have a 2 then you can downgrade to 112 - 119


the lowest version with kernver 3 is 120 - latest

**Reply by FunnyCorgi** [reply]
I am kernver 2. I was able to get to 112, but it said "[my organization name hidden for privacy reasons] requires you to update."

What I am saying is can I utilize the fact that managed chromebooks can go to any of the past three versions, to go to first 115, then 112.
